californium-core/src/main/java/org/eclipse/californium/core/coap/OptionSet.java

```diff
@@ -167,9 +168,9 @@ public OptionSet(OptionSet origin) {
 		if (origin.block2 != null)
 			block2          = new BlockOption(origin.block2);
 		
-		observe = origin.observe;
+		observe = origin.observe;	
```

Comment:
Please, not source format changes without code changes. So, please revert this change.

Reply:
Of course. They sneak in sometimes ;)